src/components/Calendar/Calendar.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import InfiniteCalendar from 'react-infinite-calendar';
+import { getJson } from '../../modules/helpers';
+import 'react-infinite-calendar/styles.css';
```

Comment:
Just out of curiosity, is there any reason why you chose this component instead of others?

It really is just a question, no catch here!!! 😝